From 50b0022d5ef1e5990df53da89235eaa56d336be9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Mon, 13 Oct 2014 07:39:27 -0700
Subject: [PATCH 01/79] vlc_fixup: Avoid redefining _ and N_

---
 include/vlc_fixups.h | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/include/vlc_fixups.h b/include/vlc_fixups.h
index cee1254..56e9880 100644
--- a/include/vlc_fixups.h
+++ b/include/vlc_fixups.h
@@ -252,8 +252,13 @@ static inline locale_t newlocale(int mask, const char * locale, locale_t base)
 #endif
 
 /* libintl support */
-#define _(str)            vlc_gettext (str)
-#define N_(str)           gettext_noop (str)
+#ifndef _
+#	define _(str)            vlc_gettext (str)
+#endif
+
+#ifndef N_
+# 	define N_(str)           gettext_noop (str)
+#endif
 #define gettext_noop(str) (str)
 
 #ifdef __cplusplus
-- 
2.2.0

