From becf75d62812ca885e61cb620297eff85d3d1a08 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Tue, 21 Oct 2014 16:26:23 +0200
Subject: [PATCH 10/96] direct3d: Remove mingw only macro

---
 modules/video_output/msw/direct3d9.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules/video_output/msw/direct3d9.c b/modules/video_output/msw/direct3d9.c
index 3ab12f1..d24eb8b 100644
--- a/modules/video_output/msw/direct3d9.c
+++ b/modules/video_output/msw/direct3d9.c
@@ -1295,14 +1295,14 @@ static int Direct3D9CompileShader(vout_display_t *vd, const char *shader_source,
     if (FAILED(hr)) {
         msg_Warn(vd, "D3DXCompileShader Error (hr=0x%0lx)", hr);
         if (error_msgs) {
-            msg_Warn(vd, "HLSL Compilation Error: %s", (char*)ID3DXBuffer_GetBufferPointer(error_msgs));
+            msg_Warn(vd, "HLSL Compilation Error: %s", (char*)error_msgs->lpVtbl->GetBufferPointer(error_msgs));
             ID3DXBuffer_Release(error_msgs);
     }
         return VLC_EGENERIC;
     }
 
     hr = IDirect3DDevice9_CreatePixelShader(sys->d3ddev,
-            ID3DXBuffer_GetBufferPointer(compiled_shader),
+            compiled_shader->lpVtbl->GetBufferPointer(compiled_shader),
             &sys->d3dx_shader);
 
     if (compiled_shader)
-- 
2.4.2

