From 936303ea9cce97bb97eba236d69a1d8569bbdcd4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Wed, 22 Oct 2014 12:27:19 +0200
Subject: [PATCH 16/71] libvlc: Use alloca instead of VLAs

---
 lib/core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/core.c b/lib/core.c
index fa5c4de..4e50904 100644
--- a/lib/core.c
+++ b/lib/core.c
@@ -46,7 +46,7 @@ libvlc_instance_t * libvlc_new( int argc, const char *const *argv )
     if (unlikely(p_new == NULL))
         return NULL;
 
-    const char *my_argv[argc + 2];
+    const char **my_argv = alloca((argc + 2) * sizeof(*my_argv));
     my_argv[0] = "libvlc"; /* dummy arg0, skipped by getopt() et al */
     for( int i = 0; i < argc; i++ )
          my_argv[i + 1] = argv[i];
-- 
2.2.0

