From ba69a9eb695e4569b02c4406c649fc8b01d8867c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Wed, 22 Oct 2014 14:14:23 +0200
Subject: [PATCH 18/99] HACK: build a52 with msvc

---
 contrib/src/a52/rules.mak | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/contrib/src/a52/rules.mak b/contrib/src/a52/rules.mak
index 1db72df..db4e79d 100644
--- a/contrib/src/a52/rules.mak
+++ b/contrib/src/a52/rules.mak
@@ -26,4 +26,7 @@ endif
 	cd $< && $(HOSTVARS) ./configure $(HOSTCONF)
 	cd $</liba52 && $(MAKE) install
 	cd $</include && $(MAKE) install
+ifdef HAVE_VISUALSTUDIO
+	cp $(PREFIX)/lib/a52.lib $(PREFIX)/lib/liba52.a
+endif
 	touch $@
-- 
1.9.5.msysgit.0

