From 418c6807fbfd067fdf75fd142d8f88edaba8956c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Wed, 22 Oct 2014 15:06:37 +0200
Subject: [PATCH 24/99] contribs: mpcdec: Build with msvc when available

---
 contrib/src/mpcdec/rules.mak | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/contrib/src/mpcdec/rules.mak b/contrib/src/mpcdec/rules.mak
index 44cab85..2f645ce 100644
--- a/contrib/src/mpcdec/rules.mak
+++ b/contrib/src/mpcdec/rules.mak
@@ -43,8 +43,15 @@ endif
 	touch $@
 
 .mpcdec: musepack toolchain.cmake
+ifdef HAVE_VISUALSTUDIO	
+	cd $< && $(HOSTVARS_PIC) $(CMAKE) -DSHARED=OFF -G "Visual Studio 12 2013 ARM" .
+	cd $< && msbuild.exe -p:Configuration=Release -p:Platform=ARM INSTALL.vcxproj
+	cd $< && cp libmpcdec/Release/mpcdec_static.lib "$(PREFIX)/lib/mpcdec.lib"
+	cd $< && cp "$(PREFIX)/lib/mpcdec.lib" "$(PREFIX)/lib/libmpcdec.a"
+else
 	cd $< && $(HOSTVARS_PIC) $(CMAKE) -DSHARED=OFF .
 	cd $< && $(MAKE) install
 	mkdir -p -- "$(PREFIX)/lib"
 	cd $< && cp libmpcdec/libmpcdec_static.a "$(PREFIX)/lib/libmpcdec.a"
+endif
 	touch $@
-- 
1.9.5.msysgit.0

