From 5bee26b112e90228cc121cd4883a4fb9dcc67a2a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Wed, 22 Oct 2014 15:04:40 +0200
Subject: [PATCH 27/75] contribs: libmpeg2: Disable ASM when building with MSVC

---
 contrib/src/libmpeg2/rules.mak | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/contrib/src/libmpeg2/rules.mak b/contrib/src/libmpeg2/rules.mak
index 650814a..e12012f 100644
--- a/contrib/src/libmpeg2/rules.mak
+++ b/contrib/src/libmpeg2/rules.mak
@@ -10,6 +10,11 @@ ifeq ($(call need_pkg,"libmpeg2"),)
 PKGS_FOUND += libmpeg2
 endif
 
+CONF=--without-x --disable-sdl
+ifdef HAVE_VISUALSTUDIO
+CONF += --disable-asm
+endif
+
 $(TARBALLS)/libmpeg2-$(LIBMPEG2_VERSION).tar.gz:
 	$(call download,$(LIBMPEG2_URL))
 
@@ -23,7 +28,7 @@ libmpeg2: libmpeg2-$(LIBMPEG2_VERSION).tar.gz .sum-libmpeg2
 	$(MOVE)
 
 .libmpeg2: libmpeg2
-	cd $< && $(HOSTVARS) ./configure $(HOSTCONF) --without-x --disable-sdl
+	cd $< && $(HOSTVARS) ./configure $(HOSTCONF) $(CONF)
 	cd $</libmpeg2 && $(MAKE) && $(MAKE) install
 	cd $</include && $(MAKE) && $(MAKE) install
 	touch $@
-- 
2.2.0

