From 6785bf6707cc5ee7d4006165f1496219dc6d9371 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Wed, 22 Oct 2014 15:30:36 +0200
Subject: [PATCH 028/129] winvlc: Use alloca rather than VLA

---
 bin/winvlc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/winvlc.c b/bin/winvlc.c
index 33784e2..ecf296a 100644
--- a/bin/winvlc.c
+++ b/bin/winvlc.c
@@ -109,7 +109,7 @@ int WINAPI WinMain( HINSTANCE hInstance, HINSTANCE hPrevInstance,
     if (wargv == NULL)
         return 1;
 
-    char *argv[argc + 3];
+    char **argv = alloca((argc + 3) * sizeof(*argv));
     BOOL crash_handling = TRUE;
     int j = 0;
     char *lang = NULL;
-- 
1.9.5.msysgit.1

