From 002ad36bb25985dc62d1a1dd0a842261f15b5e76 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Wed, 22 Oct 2014 15:06:37 +0200
Subject: [PATCH 29/62] HACK: contribs: mpcdec

---
 contrib/src/mpcdec/rules.mak | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/contrib/src/mpcdec/rules.mak b/contrib/src/mpcdec/rules.mak
index e0c004a..8e7f364 100644
--- a/contrib/src/mpcdec/rules.mak
+++ b/contrib/src/mpcdec/rules.mak
@@ -40,8 +40,14 @@ endif
 	touch $@
 
 .mpcdec: musepack toolchain.cmake
-	cd $< && $(HOSTVARS_PIC) $(CMAKE) -DSHARED=OFF .
+	cd $< && $(HOSTVARS_PIC) $(CMAKE) -DSHARED=OFF . -G "Visual Studio 12 2013 ARM"
+ifdef HAVE_VISUALSTUDIO	
+	cd $< && msbuild.exe -p:Configuration=Release -p:Platform=ARM INSTALL.vcxproj
+	cd $< && cp libmpcdec/Release/mpcdec_static.lib "$(PREFIX)/lib/mpcdec.lib"
+	cd $< && cp "$(PREFIX)/lib/mpcdec.lib" "$(PREFIX)/lib/libmpcdec.a"
+else
 	cd $< && $(MAKE) install
 	mkdir -p -- "$(PREFIX)/lib"
 	cd $< && cp libmpcdec/libmpcdec_static.a "$(PREFIX)/lib/libmpcdec.a"
+endif
 	touch $@
-- 
2.2.0

