From 7fa89bcbce51b56a4e5511d809fac2ef4479e0fe Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Wed, 22 Oct 2014 15:30:36 +0200
Subject: [PATCH 33/79] winvlc: Use alloca rather than VLA

---
 bin/winvlc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/winvlc.c b/bin/winvlc.c
index 7caab51..bd64488 100644
--- a/bin/winvlc.c
+++ b/bin/winvlc.c
@@ -109,7 +109,7 @@ int WINAPI WinMain( HINSTANCE hInstance, HINSTANCE hPrevInstance,
     if (wargv == NULL)
         return 1;
 
-    char *argv[argc + 4];
+    char **argv = alloca((argc + 4) * sizeof(*argv));
     BOOL crash_handling = TRUE;
     int j = 0;
     char *lang = NULL;
-- 
2.2.0

