From 09da25929f1d886067601a065a48a0baf8b2b8c4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Wed, 22 Oct 2014 15:43:53 +0200
Subject: [PATCH 34/76] winvlc: Fix build issue with multiline strings &
 unicode macros/prefixes

---
 bin/winvlc.c | 46 +++++++++++++++++++++++-----------------------
 1 file changed, 23 insertions(+), 23 deletions(-)

diff --git a/bin/winvlc.c b/bin/winvlc.c
index bd64488..c37f2b1 100644
--- a/bin/winvlc.c
+++ b/bin/winvlc.c
@@ -191,8 +191,8 @@ int WINAPI WinMain( HINSTANCE hInstance, HINSTANCE hPrevInstance,
         libvlc_release (vlc);
     }
     else
-        MessageBox (NULL, TEXT("VLC media player could not start.\n"
-                    "Either the command line options were invalid or no plugins were found.\n"),
+        MessageBox (NULL, TEXT("VLC media player could not start.\n")
+                    TEXT("Either the command line options were invalid or no plugins were found.\n"),
                     TEXT("VLC media player"),
                     MB_OK|MB_ICONERROR);
 
@@ -219,8 +219,8 @@ static void check_crashdump(void)
         return;
     fclose( fd );
 
-    int answer = MessageBox( NULL, L"Ooops: VLC media player just crashed.\n" \
-    "Would you like to send a bug report to the developers team?",
+    int answer = MessageBox( NULL, L"Ooops: VLC media player just crashed.\n"
+    L"Would you like to send a bug report to the developers team?",
     L"VLC crash reporting", MB_YESNO);
 
     if(answer == IDYES)
@@ -244,20 +244,20 @@ static void check_crashdump(void)
 
                 if( FtpPutFile( ftp, mv_crashdump_path, remote_file,
                             FTP_TRANSFER_TYPE_BINARY, 0) )
-                    MessageBox( NULL, L"Report sent correctly. Thanks a lot " \
-                                "for the help.", L"Report sent", MB_OK);
+                    MessageBox( NULL, L"Report sent correctly. Thanks a lot for the help.", 
+                    L"Report sent", MB_OK);
                 else
-                    MessageBox( NULL, L"There was an error while "\
-                                "transferring the data to the FTP server.\n"\
-                                "Thanks a lot for the help.",
+                    MessageBox( NULL, L"There was an error while "
+                                L"transferring the data to the FTP server.\n"
+                                L"Thanks a lot for the help.",
                                 L"Report sending failed", MB_OK);
                 InternetCloseHandle(ftp);
             }
             else
             {
-                MessageBox( NULL, L"There was an error while connecting to " \
-                                "the FTP server. "\
-                                "Thanks a lot for the help.",
+                MessageBox( NULL, L"There was an error while connecting to "
+                                L"the FTP server. "
+                                L"Thanks a lot for the help.",
                                 L"Report sending failed", MB_OK);
                 fprintf(stderr,"Can't connect to FTP server 0x%08lu\n",
                         (unsigned long)GetLastError());
@@ -266,8 +266,8 @@ static void check_crashdump(void)
         }
         else
         {
-              MessageBox( NULL, L"There was an error while connecting to the Internet.\n"\
-                                "Thanks a lot for the help anyway.",
+              MessageBox( NULL, L"There was an error while connecting to the Internet.\n"
+                                L"Thanks a lot for the help anyway.",
                                 L"Report sending failed", MB_OK);
         }
     }
@@ -303,7 +303,7 @@ LONG WINAPI vlc_exception_filter(struct _EXCEPTION_POINTERS *lpExceptionInfo)
         osvi.dwOSVersionInfoSize = sizeof( OSVERSIONINFO );
         GetVersionEx( &osvi );
 
-        fwprintf( fd, L"[version]\nOS=%d.%d.%d.%d.%ls\nVLC=" VERSION_MESSAGE,
+        fwprintf( fd, L"[version]\nOS=%d.%d.%d.%d.%ls\nVLC=" TEXT(VERSION_MESSAGE),
                 osvi.dwMajorVersion, osvi.dwMinorVersion, osvi.dwBuildNumber,
                 osvi.dwPlatformId, osvi.szCSDVersion);
 
@@ -319,11 +319,11 @@ LONG WINAPI vlc_exception_filter(struct _EXCEPTION_POINTERS *lpExceptionInfo)
             fwprintf( fd, L" | %p", pException->ExceptionInformation[i] );
 
 #ifdef _WIN64
-        fwprintf( fd, L"\n\n[context]\nRDI:%px\nRSI:%px\n" \
-                    "RBX:%px\nRDX:%px\nRCX:%px\nRAX:%px\n" \
-                    "RBP:%px\nRIP:%px\nRSP:%px\nR8:%px\n" \
-                    "R9:%px\nR10:%px\nR11:%px\nR12:%px\n" \
-                    "R13:%px\nR14:%px\nR15:%px\n",
+        fwprintf( fd, L"\n\n[context]\nRDI:%px\nRSI:%px\n"
+                    L"RBX:%px\nRDX:%px\nRCX:%px\nRAX:%px\n"
+                    L"RBP:%px\nRIP:%px\nRSP:%px\nR8:%px\n"
+                    L"R9:%px\nR10:%px\nR11:%px\nR12:%px\n"
+                    L"R13:%px\nR14:%px\nR15:%px\n",
                         pContext->Rdi,pContext->Rsi,pContext->Rbx,
                         pContext->Rdx,pContext->Rcx,pContext->Rax,
                         pContext->Rbp,pContext->Rip,pContext->Rsp,
@@ -331,9 +331,9 @@ LONG WINAPI vlc_exception_filter(struct _EXCEPTION_POINTERS *lpExceptionInfo)
                         pContext->R11,pContext->R12,pContext->R13,
                         pContext->R14,pContext->R15 );
 #else
-        fwprintf( fd, L"\n\n[context]\nEDI:%px\nESI:%px\n" \
-                    "EBX:%px\nEDX:%px\nECX:%px\nEAX:%px\n" \
-                    "EBP:%px\nEIP:%px\nESP:%px\n",
+        fwprintf( fd, L"\n\n[context]\nEDI:%px\nESI:%px\n"
+                    L"EBX:%px\nEDX:%px\nECX:%px\nEAX:%px\n"
+                    L"EBP:%px\nEIP:%px\nESP:%px\n",
                         pContext->Edi,pContext->Esi,pContext->Ebx,
                         pContext->Edx,pContext->Ecx,pContext->Eax,
                         pContext->Ebp,pContext->Eip,pContext->Esp );
-- 
2.2.0

