From 0de1b97dbdecb2a505cefee98c09c94239816b7d Mon Sep 17 00:00:00 2001
From: Jean-Baptiste Kempf <jb@videolan.org>
Date: Wed, 16 Apr 2014 16:09:53 +0200
Subject: [PATCH 35/77] Cheat for Windows Store subtitles

---
 lib/video.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/video.c b/lib/video.c
index 4abe36e..7ea21a8 100644
--- a/lib/video.c
+++ b/lib/video.c
@@ -350,7 +350,7 @@ int libvlc_video_set_subtitle_file( libvlc_media_player_t *p_mi,
 
     if( p_input_thread )
     {
-        if( !input_AddSubtitle( p_input_thread, psz_subtitle, true ) )
+        if( !input_AddSubtitle( p_input_thread, psz_subtitle, false) )
             b_ret = true;
         vlc_object_release( p_input_thread );
     }
-- 
2.2.0

