From 87d586c8be3573e4a8fcdacf4c89f98252ddd5d2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Wed, 22 Oct 2014 15:54:42 +0200
Subject: [PATCH 35/75] cachegen: Quick & dirty fix whtn building without
 getopt

---
 bin/cachegen.c | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/bin/cachegen.c b/bin/cachegen.c
index 8e72bf4..b3eb34d 100644
--- a/bin/cachegen.c
+++ b/bin/cachegen.c
@@ -29,7 +29,7 @@
 #ifdef HAVE_GETOPT_H
 # include <getopt.h>
 #endif
-#ifdef WIN32
+#ifdef _WIN32
 # include <windows.h>
 #endif
 
@@ -49,9 +49,10 @@ static void usage (const char *path)
 
 int main (int argc, char *argv[])
 {
-#ifdef WIN32
+#ifdef _WIN32
     SetErrorMode(SEM_FAILCRITICALERRORS);
 #endif
+#ifdef HAVE_GETOPT_H
     static const struct option opts[] =
     {
         { "force",      no_argument,       NULL, 'f' },
@@ -79,6 +80,10 @@ int main (int argc, char *argv[])
                 usage (argv[0]);
                 return 1;
         }
+#else
+    int optind = 1;
+    bool force = true;
+#endif
 
     for (int i = optind; i < argc; i++)
     {
-- 
2.2.0

