From d9359e741f9b41b43218dc8215b57afeb4b5003d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Wed, 22 Oct 2014 15:56:03 +0200
Subject: [PATCH 36/55] modules: bank: Workaround lack of "lib" prefix

---
 src/modules/bank.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/bank.c b/src/modules/bank.c
index f4fba50..0b3c177 100644
--- a/src/modules/bank.c
+++ b/src/modules/bank.c
@@ -474,7 +474,7 @@ static void AllocatePluginDir (module_bank_t *bank, unsigned maxdepth,
             static const char suffix[] = "_plugin"LIBEXT;
             size_t len = strlen (file);
 
-#ifndef __OS2__
+#if !defined(__OS2__) && !defined(VLC_WINSTORE_APP)
             /* Check that file matches the "lib*_plugin"LIBEXT pattern */
             if (len > strlen (suffix)
              && !strncmp (file, prefix, strlen (prefix))
-- 
2.1.1

