From 80170737baa75ef3ace5e532e4551f9845704e48 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Thu, 20 Nov 2014 15:42:35 +0100
Subject: [PATCH 037/128] contribs: iconv: Fix msvc build

---
 contrib/src/iconv/rules.mak | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/contrib/src/iconv/rules.mak b/contrib/src/iconv/rules.mak
index 3596463..9826915 100644
--- a/contrib/src/iconv/rules.mak
+++ b/contrib/src/iconv/rules.mak
@@ -28,6 +28,7 @@ ifdef HAVE_IOS
 	$(APPLY) $(SRC)/iconv/libiconv-android-ios.patch
 endif
 ifdef HAVE_WINRT
+	$(APPLY) $(SRC)/iconv/libiconv-android-ios.patch
 	$(APPLY) $(SRC)/iconv/libiconv-winrt.patch
 endif
 	$(UPDATE_AUTOCONFIG) && cd $(UNPACK_DIR) && mv config.guess config.sub build-aux
@@ -37,4 +38,8 @@ endif
 .iconv: iconv
 	cd $< && $(HOSTVARS) ./configure $(HOSTCONF) --disable-nls
 	cd $< && $(MAKE) install
+ifdef HAVE_VISUALSTUDIO
+	# VLC doesn't use libtool values to test linking libs
+	cp $(PREFIX)/lib/iconv.lib $(PREFIX)/lib/libiconv.a
+endif
 	touch $@
-- 
1.9.5.msysgit.1

