From e270b85722ab97c6204209caf7fefe639a7cb769 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Thu, 20 Nov 2014 16:21:27 +0100
Subject: [PATCH 040/128] contribs: fontconfig: Always configure before build

---
 contrib/src/fontconfig/rules.mak | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/src/fontconfig/rules.mak b/contrib/src/fontconfig/rules.mak
index 78b92c7..20eb967 100644
--- a/contrib/src/fontconfig/rules.mak
+++ b/contrib/src/fontconfig/rules.mak
@@ -49,12 +49,12 @@ endif
 DEPS_fontconfig = freetype2 $(DEPS_freetype2) libxml2 $(DEPS_libxml2)
 
 .fontconfig: fontconfig
+	cd $< && $(FONTCONFIG_ENV) ./configure $(FONTCONFIG_CONF)
 ifdef HAVE_WIN32
 	cd $< && $(MAKE) -C src install
 	sed -e 's%/usr/lib/libiconv.la%%' -i.orig $(PREFIX)/lib/libfontconfig.la
 	cp $</fontconfig.pc $(PREFIX)/lib/pkgconfig/
 else
-	cd $< && $(FONTCONFIG_ENV) ./configure $(FONTCONFIG_CONF)
 	cd $< && $(MAKE)
 ifndef HAVE_MACOSX
 	cd $< && $(MAKE) install
-- 
1.9.5.msysgit.1

