From 4d3ba21d778f46891c7f526951058182d7e6d2ea Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Thu, 20 Nov 2014 16:40:17 +0100
Subject: [PATCH 44/74] contribs: speedxdsp: Disable ASM on windows phone

---
 contrib/src/speexdsp/rules.mak | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/contrib/src/speexdsp/rules.mak b/contrib/src/speexdsp/rules.mak
index e82c6ba..13065c9 100644
--- a/contrib/src/speexdsp/rules.mak
+++ b/contrib/src/speexdsp/rules.mak
@@ -25,6 +25,9 @@ endif
 ifndef HAVE_NEON
 SPEEXDSP_CONF += --disable-neon
 endif
+ifdef HAVE_WINPHONE
+SPEEXDSP_CONF += --disable-neon
+endif
 ifndef HAVE_FPU
 SPEEXDSP_CONF += --enable-fixed-point
 ifeq ($(ARCH),arm)
-- 
2.2.0

