From f526fff38a397dc17e8b477030ca7bc9eb0359b0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Fri, 21 Nov 2014 13:12:36 +0100
Subject: [PATCH 45/99] mft: Fix build with MSVC

---
 modules/codec/mft.c | 24 ++++++++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/modules/codec/mft.c b/modules/codec/mft.c
index 8c9373a..f88af01 100644
--- a/modules/codec/mft.c
+++ b/modules/codec/mft.c
@@ -63,6 +63,30 @@ vlc_module_begin()
     set_subcategory(SUBCAT_INPUT_VCODEC)
 vlc_module_end()
 
+#ifdef VLC_WINSTORE_APP
+typedef struct __WIDL_mfobjects_generated_name_00000016 {
+    GUID guidMajorType;
+    GUID guidSubtype;
+} MFT_REGISTER_TYPE_INFO;
+
+enum _MFT_ENUM_FLAG {
+    MFT_ENUM_FLAG_SYNCMFT = 0x00000001,
+    MFT_ENUM_FLAG_ASYNCMFT = 0x00000002,
+    MFT_ENUM_FLAG_HARDWARE = 0x00000004,
+    MFT_ENUM_FLAG_FIELDOFUSE = 0x00000008,
+    MFT_ENUM_FLAG_LOCALMFT = 0x00000010,
+    MFT_ENUM_FLAG_TRANSCODE_ONLY = 0x00000020,
+    MFT_ENUM_FLAG_SORTANDFILTER = 0x00000040,
+    MFT_ENUM_FLAG_SORTANDFILTER_APPROVED_ONLY = 0x000000c0,
+    MFT_ENUM_FLAG_SORTANDFILTER_WEB_ONLY = 0x00000140,
+    MFT_ENUM_FLAG_ALL = 0x0000003f
+  };
+
+DEFINE_GUID (MF_MT_ORIGINAL_WAVE_FORMAT_TAG, 0x8cbbc843, 0x9fd9, 0x49c2, 0x88, 0x2f, 0xa7, 0x25, 0x86, 0xc4, 0x08, 0xad);
+
+STDAPI MFTEnumEx (GUID guidCategory, UINT32 Flags, const MFT_REGISTER_TYPE_INFO *pInputType, const MFT_REGISTER_TYPE_INFO *pOutputType, IMFActivate ***pppMFTActivate, UINT32 *pnumMFTActivate);
+#endif
+
 typedef struct
 {
     HINSTANCE mfplat_dll;
-- 
1.9.5.msysgit.1

