From 7de935d0ba0926fccb028981971ec3f8b9a2d204 Mon Sep 17 00:00:00 2001
From: Jean-Baptiste Kempf <jb@videolan.org>
Date: Thu, 20 Nov 2014 13:12:05 +0100
Subject: [PATCH 46/99] Fix gai_strerror on Unicode

---
 include/vlc_network.h | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/include/vlc_network.h b/include/vlc_network.h
index c66e2d1..d297ad3 100644
--- a/include/vlc_network.h
+++ b/include/vlc_network.h
@@ -38,6 +38,13 @@
 #   include <winsock2.h>
 #   include <ws2tcpip.h>
 #   define net_errno (WSAGetLastError())
+#   undef gai_strerror
+#   ifdef UNICODE
+#       include <vlc_charset.h>
+#       define gai_strerror(a) FromWide(gai_strerrorW)
+#   else
+#       define gai_strerror gai_strerrorA
+#   endif
 
 struct iovec
 {
@@ -229,11 +236,6 @@ VLC_API ssize_t net_vaPrintf( vlc_object_t *p_this, int fd, const char *psz_fmt,
 # define AI_IDN 0 /* GNU/libc extension */
 #endif
 
-#ifdef _WIN32
-# undef gai_strerror
-# define gai_strerror gai_strerrorA
-#endif
-
 #ifdef __OS2__
 # ifndef NI_NUMERICHOST
 #  define NI_NUMERICHOST 0x01
-- 
1.9.5.msysgit.1

