From a2b320ea11aee8c76170ea0798123808b1cc3a89 Mon Sep 17 00:00:00 2001
From: Jean-Baptiste Kempf <jb@videolan.org>
Date: Thu, 20 Nov 2014 13:12:05 +0100
Subject: [PATCH 047/128] Fix gai_strerror on Unicode

---
 include/vlc_network.h | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/include/vlc_network.h b/include/vlc_network.h
index 4da4faf..c5296c4 100644
--- a/include/vlc_network.h
+++ b/include/vlc_network.h
@@ -221,7 +221,12 @@ VLC_API ssize_t net_vaPrintf( vlc_object_t *p_this, int fd, const char *psz_fmt,
 
 #ifdef _WIN32
 # undef gai_strerror
-# define gai_strerror gai_strerrorA
+# ifdef UNICODE
+#   include <vlc_charset.h>
+#   define gai_strerror(a) FromWide(gai_strerrorW)
+# else
+#   define gai_strerror gai_strerrorA
+# endif
 #endif
 
 #ifdef __OS2__
-- 
1.9.5.msysgit.1

