From 79d9edbea69ee0d4c6894a71f3a1b462f349e5ae Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Wed, 7 Jan 2015 11:27:53 +0100
Subject: [PATCH 053/128] contribs: taglib: Use visual studio when available

---
 contrib/src/taglib/rules.mak | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/contrib/src/taglib/rules.mak b/contrib/src/taglib/rules.mak
index 055b6cf..ad8b909 100644
--- a/contrib/src/taglib/rules.mak
+++ b/contrib/src/taglib/rules.mak
@@ -24,5 +24,10 @@ taglib: taglib-$(TAGLIB_VERSION).tar.gz .sum-taglib
 		-DENABLE_STATIC:BOOL=ON \
 		-DWITH_ASF:BOOL=ON \
 		-DWITH_MP4:BOOL=ON .
+ifdef HAVE_VISUALSTUDIO
+	cd $< && msbuild.exe -p:Configuration=$(VLC_CONFIGURATION) -m -nologo INSTALL.vcxproj
+	cd $< && cp taglib/$(VLC_CONFIGURATION)/tag.lib "$(PREFIX)/lib/libtag.a"
+else
 	cd $< && $(MAKE) install
+endif
 	touch $@
-- 
1.9.5.msysgit.1

