From 872b0dae7ccf9d287123a1c990426cb3c717d9f9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Thu, 5 Feb 2015 11:14:54 +0100
Subject: [PATCH 060/129] HACK HACK HACK: Disable dmo module

---
 modules/codec/Makefile.am | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules/codec/Makefile.am b/modules/codec/Makefile.am
index 4ae29ac..38fb63b 100644
--- a/modules/codec/Makefile.am
+++ b/modules/codec/Makefile.am
@@ -519,8 +519,10 @@ libqsv_plugin_la_CPPFLAGS = $(AM_CPPFLAGS)
 libdmo_plugin_la_LIBADD = $(LIBM)
 if HAVE_WIN32
 libdmo_plugin_la_LIBADD += -lole32 -luuid
+if !HAVE_WINSTORE
 codec_LTLIBRARIES += libdmo_plugin.la
 endif
+endif
 
 libmft_plugin_la_SOURCES = codec/mft.c packetizer/h264_nal.c packetizer/h264_nal.h
 if HAVE_WIN32
-- 
1.9.5.msysgit.1

