From 2929ed331717712adeb2d2953ecb372d90f76b81 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Wed, 22 Oct 2014 15:54:42 +0200
Subject: [PATCH 062/128] cachegen: Quick & dirty fix whtn building without
 getopt

---
 bin/cachegen.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/bin/cachegen.c b/bin/cachegen.c
index 25ea925..39b3f1d 100644
--- a/bin/cachegen.c
+++ b/bin/cachegen.c
@@ -51,6 +51,7 @@ int main (int argc, char *argv[])
 #ifdef WIN32
     SetErrorMode(SEM_FAILCRITICALERRORS);
 #endif
+#ifdef HAVE_GETOPT_H
     static const struct option opts[] =
     {
         { "help",       no_argument,       NULL, 'h' },
@@ -73,6 +74,10 @@ int main (int argc, char *argv[])
                 usage (argv[0]);
                 return 1;
         }
+#else
+    int optind = 1;
+    bool force = true;
+#endif
 
     for (int i = optind; i < argc; i++)
     {
-- 
1.9.5.msysgit.1

