From 3e8e00bb0b6f9179d6288fb7eb139671e79f81e0 Mon Sep 17 00:00:00 2001
From: Jean-Baptiste Kempf <jb@videolan.org>
Date: Wed, 25 Feb 2015 10:14:43 +0100
Subject: [PATCH 063/128] Fix mft

---
 modules/codec/mft.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/codec/mft.c b/modules/codec/mft.c
index f88af01..79647bc 100644
--- a/modules/codec/mft.c
+++ b/modules/codec/mft.c
@@ -1116,7 +1116,7 @@ static int FindMFT(decoder_t *p_dec)
 
 static int LoadMFTLibrary(MFHandle *mf)
 {
-#if _WIN32_WINNT < 0x601
+#if _WIN32_WINNT < 0x601 || !defined(_MSC_VER)
     mf->mfplat_dll = LoadLibrary(TEXT("mfplat.dll"));
     if (!mf->mfplat_dll)
         return VLC_EGENERIC;
-- 
1.9.5.msysgit.1

