From 9ac5360be67a9964f1747b879db781b3a824a4ea Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Thu, 12 Feb 2015 15:56:05 +0100
Subject: [PATCH 066/128] HACK: contribs: matroska: Force library name

---
 contrib/src/matroska/rules.mak | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/contrib/src/matroska/rules.mak b/contrib/src/matroska/rules.mak
index c71ee83..c29b093 100644
--- a/contrib/src/matroska/rules.mak
+++ b/contrib/src/matroska/rules.mak
@@ -21,4 +21,7 @@ MATROSKA_EXTRA_FLAGS = CXXFLAGS="${CXXFLAGS} -fvisibility=hidden"
 .matroska: libmatroska
 	cd $< && $(HOSTVARS) ./configure $(HOSTCONF) $(MATROSKA_EXTRA_FLAGS)
 	cd $< && $(MAKE) install
+ifdef HAVE_VISUALSTUDIO
+	cp $(PREFIX)/lib/matroska.lib $(PREFIX)/lib/libmatroska.a
+endif
 	touch $@
-- 
1.9.5.msysgit.1

