From bf425d300d780219679b6d2fe9ccd264b8226ab5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Thu, 5 Feb 2015 11:18:25 +0100
Subject: [PATCH 73/82] HACK HACK HACK: Disable mft

---
 modules/codec/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/codec/Makefile.am b/modules/codec/Makefile.am
index 3d7f7df..47af3f9 100644
--- a/modules/codec/Makefile.am
+++ b/modules/codec/Makefile.am
@@ -501,7 +501,7 @@ endif
 libmft_plugin_la_SOURCES = codec/mft.c codec/h264_nal.c codec/h264_nal.h
 if HAVE_WIN32
 libmft_plugin_la_LIBADD = -lole32 -luuid -lmfuuid -lmfplat
-codec_LTLIBRARIES += libmft_plugin.la
+# codec_LTLIBRARIES += libmft_plugin.la
 endif
 
 libquicktime_plugin_la_SOURCES = codec/quicktime.c
-- 
2.2.0

