From 2eb19f309fe5b9ea1909eb59fe9ece9f386443ce Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Thu, 5 Feb 2015 11:14:54 +0100
Subject: [PATCH 75/78] HACK HACK HACK: Disable dmo module

---
 modules/codec/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/codec/Makefile.am b/modules/codec/Makefile.am
index 0de32a0..3d7f7df 100644
--- a/modules/codec/Makefile.am
+++ b/modules/codec/Makefile.am
@@ -495,7 +495,7 @@ libqsv_plugin_la_CPPFLAGS = $(AM_CPPFLAGS)
 libdmo_plugin_la_LIBADD = $(LIBM)
 if HAVE_WIN32
 libdmo_plugin_la_LIBADD += -lole32 -luuid
-codec_LTLIBRARIES += libdmo_plugin.la
+# codec_LTLIBRARIES += libdmo_plugin.la
 endif
 
 libmft_plugin_la_SOURCES = codec/mft.c codec/h264_nal.c codec/h264_nal.h
-- 
2.2.0

