From 7a1b0a6784dfa60b8e21eb5a99c230461659f939 Mon Sep 17 00:00:00 2001
From: Steve Lhomme <robUx4@gmail.com>
Date: Mon, 3 Aug 2015 09:50:11 +0200
Subject: [PATCH 076/128] ignore the MSVC generated contribs

---
 contrib/.gitignore | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/contrib/.gitignore b/contrib/.gitignore
index e51bc0f..4420a0b 100644
--- a/contrib/.gitignore
+++ b/contrib/.gitignore
@@ -1,2 +1,4 @@
 vlc-contrib-*.tar.xz
 /*-*-*/
+/*-*-*_*/
+/*_*_*/
-- 
1.9.5.msysgit.1

