From 814dd173fa28e2902f51cddd00c8c92f697ae536 Mon Sep 17 00:00:00 2001
From: Steve Lhomme <robux4@videolabs.io>
Date: Wed, 24 Jun 2015 17:26:02 +0200
Subject: [PATCH 79/99] gitignore: ignore the build directories

---
 .gitignore | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/.gitignore b/.gitignore
index 53b8f4f..cb31048 100644
--- a/.gitignore
+++ b/.gitignore
@@ -53,3 +53,7 @@ build-*
 contrib-*
 install-*
 
+/contrib/Windows*
+/contrib/Metrox86*
+/Windows*
+/Metrox86*
-- 
1.9.5.msysgit.1

