From ca40ff742b08074c78a5979b43b4849add60f244 Mon Sep 17 00:00:00 2001
From: Steve Lhomme <robUx4@gmail.com>
Date: Thu, 6 Aug 2015 16:19:32 +0200
Subject: [PATCH 088/128] fix librotate compilation with MSVC

---
 modules/video_filter/Makefile.am | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/modules/video_filter/Makefile.am b/modules/video_filter/Makefile.am
index 79c3897..e517169 100644
--- a/modules/video_filter/Makefile.am
+++ b/modules/video_filter/Makefile.am
@@ -50,7 +50,11 @@ libpuzzle_plugin_la_LIBADD = $(LIBM)
 libripple_plugin_la_SOURCES = video_filter/ripple.c
 libripple_plugin_la_LIBADD = $(LIBM)
 librotate_plugin_la_SOURCES = video_filter/rotate.c
+if HAVE_WINSTORE
+librotate_plugin_la_SOURCES += control/motionlib.c control/motionlib.h
+else
 librotate_plugin_la_LIBADD = libvlc_motion.la $(LIBM)
+endif
 librotate_plugin_la_LDFLAGS = $(AM_LDFLAGS) -rpath '$(video_filterdir)'
 if HAVE_DARWIN
 librotate_plugin_la_LDFLAGS += -Wl,-framework,IOKit,-framework,CoreFoundation
-- 
1.9.5.msysgit.1

