From 6b0641dd14d69900b1707e8c4a2dc53b123b7a53 Mon Sep 17 00:00:00 2001
From: Steve Lhomme <robUx4@gmail.com>
Date: Thu, 6 Aug 2015 16:18:10 +0200
Subject: [PATCH 090/128] fix some header definitions

---
 include/vlc_fixups.h | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/include/vlc_fixups.h b/include/vlc_fixups.h
index aba12b3..5657b86 100644
--- a/include/vlc_fixups.h
+++ b/include/vlc_fixups.h
@@ -132,7 +132,7 @@ int vasprintf (char **, const char *, va_list);
 int ffsll(unsigned long long);
 #endif
 
-#ifndef HAVE_STRCASECMP
+#if !defined(HAVE_STRCASECMP) && !defined(strcasecmp)
 int strcasecmp (const char *, const char *);
 #endif
 
@@ -140,7 +140,9 @@ int strcasecmp (const char *, const char *);
 char *strcasestr (const char *, const char *);
 #endif
 
-#ifndef HAVE_STRDUP
+#if defined(_MSC_VER) && _MSC_VER >= 1900
+#define strdup(x) _strdup(x)
+#elif !defined(HAVE_STRDUP)
 char *strdup (const char *);
 #endif
 
@@ -184,7 +186,7 @@ lldiv_t lldiv (long long, long long);
 #endif
 
 #ifndef HAVE_STRTOF
-#ifndef __ANDROID__
+#if !defined(__ANDROID__) && defined(_MSC_VER) && _MSC_VER < 1900
 float strtof (const char *, char **);
 #endif
 #endif
@@ -415,7 +417,7 @@ void freeaddrinfo (struct addrinfo *res);
 
 /* math.h */
 
-#ifndef HAVE_NANF
+#if !defined(HAVE_NANF) && defined(_MSC_VER) && _MSC_VER < 1900
 #define nanf(tagp) NAN
 #endif
 
-- 
1.9.5.msysgit.1

