From a354ca8eae15a75377bb4c16919ccb63010d5bf3 Mon Sep 17 00:00:00 2001
From: Steve Lhomme <robux4@gmail.com>
Date: Thu, 25 Jun 2015 13:59:30 +0200
Subject: [PATCH 91/96] compat: if HAVE_STRLEN is forced we don't need to
 define the function

---
 compat/strnlen.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/compat/strnlen.c b/compat/strnlen.c
index d346681..1d8b628 100644
--- a/compat/strnlen.c
+++ b/compat/strnlen.c
@@ -24,8 +24,10 @@
 
 #include <string.h>
 
+#ifndef HAVE_STRNLEN
 size_t strnlen (const char *str, size_t max)
 {
     const char *end = memchr (str, 0, max);
     return end ? (size_t)(end - str) : max;
 }
+#endif
-- 
2.4.2

