From c7523e14ed2eb22d03b2223c5b59da26e9120253 Mon Sep 17 00:00:00 2001
From: Steve Lhomme <robux4@gmail.com>
Date: Thu, 25 Jun 2015 14:16:29 +0200
Subject: [PATCH 92/99] rand: link properly to RoGetActivationFactory

---
 src/win32/rand.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/win32/rand.c b/src/win32/rand.c
index 6e1564f..18f0e7b 100644
--- a/src/win32/rand.c
+++ b/src/win32/rand.c
@@ -33,6 +33,7 @@
 # include <windows.security.cryptography.h>
 # ifdef _MSC_VER
 #  ifndef __cplusplus
+DECLSPEC_IMPORT _Check_return_ HRESULT WINAPI RoGetActivationFactory(_In_ HSTRING activatableClassId, _In_ REFIID iid,  _COM_Outptr_ void ** factory);
 typedef __x_ABI_CWindows_CSecurity_CCryptography_CICryptographicBufferStatics ICryptographicBufferStatics;
 typedef __x_ABI_CWindows_CStorage_CStreams_CIBuffer IBuffer;
 #   define ICryptographicBufferStatics_Release __x_ABI_CWindows_CSecurity_CCryptography_CICryptographicBufferStatics_Release
-- 
1.9.5.msysgit.1

