From d3dee62c8f97590108cd3f137806e1ded065ef5b Mon Sep 17 00:00:00 2001
From: Steve Lhomme <robUx4@gmail.com>
Date: Mon, 20 Jul 2015 14:00:35 +0200
Subject: [PATCH 100/128] vlc_variables: fix a cast warning in
 var_CountChoices()

---
 include/vlc_variables.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/vlc_variables.h b/include/vlc_variables.h
index d855c30..ad76ed3 100644
--- a/include/vlc_variables.h
+++ b/include/vlc_variables.h
@@ -592,7 +592,7 @@ static inline int var_CountChoices( vlc_object_t *p_obj, const char *psz_name )
     vlc_value_t count;
     if( var_Change( p_obj, psz_name, VLC_VAR_CHOICESCOUNT, &count, NULL ) )
         return 0;
-    return count.i_int;
+    return (int) count.i_int;
 }
 #define var_CountChoices(a,b) var_CountChoices( VLC_OBJECT(a),b)
 
-- 
1.9.5.msysgit.1

