From 8ce45173c8efc632bf57a87715f72cfe9fdeb117 Mon Sep 17 00:00:00 2001
From: Steve Lhomme <robUx4@gmail.com>
Date: Thu, 13 Aug 2015 15:53:43 +0200
Subject: [PATCH 108/128] direct3d11: code cleaning

---
 modules/video_output/msw/direct3d11.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/modules/video_output/msw/direct3d11.c b/modules/video_output/msw/direct3d11.c
index 89bb04d..359f8eb 100644
--- a/modules/video_output/msw/direct3d11.c
+++ b/modules/video_output/msw/direct3d11.c
@@ -1142,13 +1142,13 @@ static void UpdatePicQuadPosition(vout_display_t *vd)
     HRESULT hr = IDXGISwapChain_GetPrivateData(sys->dxgiswapChain, &GUID_SWAPCHAIN_WIDTH, &dataSize, &i_width);
     if (FAILED(hr)) {
         msg_Err(vd, "Can't get swapchain width, size %d. (hr=0x%lX)", hr, dataSize);
-        return hr;
+        return;
     }
     dataSize = sizeof(i_height);
     hr = IDXGISwapChain_GetPrivateData(sys->dxgiswapChain, &GUID_SWAPCHAIN_HEIGHT, &dataSize, &i_height);
     if (FAILED(hr)) {
         msg_Err(vd, "Can't get swapchain height, size %d. (hr=0x%lX)", hr, dataSize);
-        return hr;
+        return;
     }
 #endif
 
@@ -1711,8 +1711,8 @@ static int Direct3D11MapSubpicture(vout_display_t *vd, int *subpicture_region_co
 
 
         /* Map the subpicture to sys->rect_dest */
-        int i_original_width  = subpicture->i_original_picture_width;
-        int i_original_height = subpicture->i_original_picture_height;
+        const int i_original_width  = subpicture->i_original_picture_width;
+        const int i_original_height = subpicture->i_original_picture_height;
 
         const RECT video = sys->rect_dest;
         const float scale_w = (float)(video.right  - video.left) / i_original_width;
-- 
1.9.5.msysgit.1

