From d14e90b01eefe3d8b22e6f59ac57a3b162eca183 Mon Sep 17 00:00:00 2001
From: Steve Lhomme <robUx4@gmail.com>
Date: Fri, 31 Jul 2015 15:30:46 +0200
Subject: [PATCH 117/128] libass: on Winstore apps we need to embed the font in
 the package

---
 modules/codec/libass.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/modules/codec/libass.c b/modules/codec/libass.c
index c67a8d9..26e2044 100644
--- a/modules/codec/libass.c
+++ b/modules/codec/libass.c
@@ -213,6 +213,9 @@ static int Create( vlc_object_t *p_this )
 #if defined( __ANDROID__ )
     const char *psz_font = "/system/fonts/DroidSans-Bold.ttf";
     const char *psz_family = "Droid Sans Bold";
+#elif VLC_WINSTORE_APP
+    const char *psz_font = "segoeui.ttf";
+    const char *psz_family = "Segoe UI";
 #else
     const char *psz_font = NULL; /* We don't ship a default font with VLC */
     const char *psz_family = "Arial"; /* Use Arial if we can't find anything more suitable */
-- 
1.9.5.msysgit.1

